build/cloud/ha/README.md

```diff
@@ -0,0 +1,160 @@
+# The HA of CloudCore(deployed in k8s cluster)
+
+**Note:**
+
+There are several ways to achieve the HA of cloudcore, for example, ingress, keepalived etc. Here we adopt the keepalived. The HA of cloudcore according to ingress will be achieved later.
+
+## Determine the virtual IP of CloudCore
+
+Determine a VIP that the CloudCore service exposed to the edge nodes. Here we recommend `keepalived` to do that. You had better directly schedule pods to specific number of nodes by `nodeSelector` when using  `keepalived`. And you have  to install `keepalived` in each of nodes where CloudCore runs. The configuration of `keepalived` is shown in the end. Here suppose the VIP is 10.10.102.242.
+
+The use of `nodeSelector` is as follow:
+
+```bash
+kubectl label nodes [nodename] [key]=[value]  # label the nodes where the cloudcore will run
+```
+
+modify the term of `nodeselector`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cloudcore
+spec:
+  template:
+    spec:
+      nodeSelector: # configure the nodeSelector here!
+        [key]: [value]
```

Comment:
please note indent

Reply:
> please note indent

I couldn't find the indent mistake here.  Could u point it out:)